ONSdigital/dp-s3
# unified all methods under a single S3 client

### What

- Unify Client and Uploader in a single S3 Client, to make it easier to use
- Split into files by functionality, and added some comments with the IAM permissions required by each file functionality.
- Improved README
- Bump to version 2

### How to review

- Make sure code changes make sense
- Make sure new file organisation and documentation make sense
- Make sure the README file is clear
- Make sure unit tests pass

### Who can review

Anyone (suggested @redhug1 )
